### PR TITLE
MaxOfVector

### PR DESCRIPTION
**Pull request by TomaszWeglinski2**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: max-of-vector
🏅 Granted 5 XP for @TomaszWeglinski2

**Comment by ziobron** [comment]
⏱ 3 XP extra for @TomaszWeglinski2 for on-time delivery

File: homework/max-of-vector/maxOfVector.hpp [hunk 1]
```diff
@@ -3,6 +3,15 @@
 #include <limits>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+
+   int max;
+   if (vec.empty()){
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:8]: Add space before `{`

File: homework/max-of-vector/maxOfVector.hpp [hunk 2]
```diff
@@ -3,6 +3,15 @@
...
+       return {};
+   }
+   for( auto el : vec){
+       if(el > max){
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:12]: Add space before `{`

File: homework/max-of-vector/maxOfVector.hpp [hunk 3]
```diff
@@ -3,6 +3,15 @@
 #include <limits>
 
 int maxOfVector(const std::vector<int>& vec) {
-    // TODO: Implement me :)
-    return {};
+
+   int max;
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:7]: Better always initialize values created on stack with default value.

File: homework/max-of-vector/maxOfVector.hpp [hunk 4]
```diff
@@ -3,6 +3,15 @@
...
+   if (vec.empty()){
+       return {};
+   }
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:10]: Redundant.

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**